app/_src/production/index.md

```diff
@@ -0,0 +1,60 @@
+
+## Overview of deployment steps
+
+Deploying {{site.mesh_product_name}} to a production environment involves the following steps:
+
+1. Decide which deployment topology you plan to use.
+1. Install kumactl.
+1. Deploy the control plane.
+1. Configure the data plane.
+1. Configure security features for {{site.mesh_product_name}}.
+
+You can read more details about these steps in the following sections.
+
+### Deployment topologies
+
+There are two deployment models that can be adopted with {{site.mesh_product_name}} in order to address any Service Mesh use-case, from the simple one running in one zone to the more complex one where multiple Kubernetes or VM zones are involved, or even hybrid universal ones where {{site.mesh_product_name}} runs simultaneously on Kubernetes and VMs.
+
+The following table describes some common use cases and the deployment modes you can use for them:
+
+| Use case | Recommended deployment mode |
+| -------- | --------------------------- |
+| You want to migrate from on-premise or virtual machines to the cloud in a brownfield project. | Multi-zone |
+| You only intend to deploy {{site.mesh_product_name}} in one zone, like one Kubernetes cluster or Amazon VPC. | Standalone |
+| You want to run a mix of Kubernetes and Universal zones. | Multi-zone |
+| You want to run workloads in different regions, clouds, and/or datacenters. | Multi-zone |
+
+### kumactl
+
+The first step after you pick your deployment mode is to install `kumactl`. `kumactl` is one of the tools you can use to access the {{site.mesh_product_name}}. `kumactl` is a CLI that can do the following:
+
+* Perform read-only operations on {{site.mesh_product_name}} resources on Kubernetes. 
+* Read and create resources in {{site.mesh_product_name}} in Universal mode.
+
+The `kumactl` binary is a client to the {{site.mesh_product_name}} HTTP API. 
+
+### Control plane and data plane architecture
+
+Once `kumactl` is installed, you can use it to configure the control plane and deploy the data plane. The control plane (CP) is never on the execution path of the requests that the services exchange with each other. It’s used as a source of truth to dynamically configure the underlying data plane proxies that are deployed alongside every instance of every service that is part of the service mesh.
+
+You can either configure a [multi-zone]<!--(/docs/{{ page.version }}/production/cp-deployment/multi-zone/)--> or <!--[standalone](/docs/{{ page.version }}/production/stand-alone/)--> control plane, depending on your organization's needs. You can deploy either a <!--[Kubernetes](/docs/{{ page.version }}/production/dp-config/dpp-on-kubernetes/)--> or <!--[Universal](/docs/{{ page.version }}/production/dp-config/dpp-on-universal/)--> data plane.
```

Comment:
Why are all of these links commented out? I'm guessing that'll change when this is done?